Makefile.am

```diff
@@ -1,6 +1,8 @@
 AUTOMAKE_OPTIONS = subdir-objects
 ACLOCAL_AMFLAGS = -I m4
 
+AM_LDFLAGS = "-pthread"
```

Comment:
Does libcec now require pthreads?

Reply:
It looks like it is if you link against libraspberrypi. Otherwise, it might not be.